cinema/serializers.py

```diff
@@ -1,6 +1,6 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, CinemaHall, Actor, Genre
 
 
 class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `ModelSerializer` for `MovieSerializer` to automatically handle model fields and relationships, such as `actors` and `genres`. This would also ensure consistency with `ActorSerializer` which uses `ModelSerializer`.